### ISSUE TITLE
NoSuchMethodError on server start-up

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#2204]
### Description
NoSuchMethodError on server start-up

### Steps to Reproduce
- Update to latest Skript 2.3.7 with backported fixes
- Start server

### Errors / Screenshots
```
[12:11:43] [Server thread/ERROR]: Error occurred while enabling Skript v2.3.7 (Is it up to date?)
java.lang.NoSuchMethodError: org.bukkit.entity.Villager$Profession.isZombie()Z
        at ch.njol.skript.entity.VillagerData.<clinit>(VillagerData.java:64) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:398) ~[?:?]
        at ch.njol.skript.SkriptAddon.loadClasses(SkriptAddon.java:108) ~[?:?]
        at ch.njol.skript.Skript.onEnable(Skript.java:419) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:352) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:416) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:458) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:372) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:439) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:258) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:764) [spigot-1.14.3.jar-2019-06-25-0522:git-Spigot-595711b-e8b39d4]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

### Server Information
* Spigot 1.14.3 git-Spigot-595711b-e8b39d4
* Skript 2.3.7 with backported fixes

**Comment by Blueyescat** [comment]
You misunderstood, 2.3.7 is for 1.13.2 and below, just with a fix from 2.4-alpha4

**Comment by SlimeDog** [comment]
OK. So back to 2.4-alpha3 on 1.14.3.

**Comment by ShaneBeee** [comment]
> OK. So back to 2.4-alpha3 on 1.14.3. Which does not have the security fix.

there was an alpha-4 release as well

**Comment by SlimeDog** [comment]
The alpha4 release is the cause of the trouble.

**Comment by ShaneBeee** [comment]
Yes and that will be fixed soon.
But this issue can be closed since 2.3.x series never supported 1.14.x
